src/main/java/gtPlusPlus/xmod/bop/blocks/BOP_Block_Registrator.java

```diff
@@ -136,31 +119,25 @@ public static final void addLogRecipes(final ItemStack aStack) {
                 } else {
                     final ItemStack tPlanks = GT_Utility.copy(tStack);
                     tPlanks.stackSize = tPlanks.stackSize * 3 / 2;
-                    GT_Values.RA.addCutterRecipe(
-                            new ItemStack(aStack.getItem(), 1, i),
-                            Materials.Lubricant.getFluid(1L),
-                            GT_Utility.copy(tPlanks),
-                            GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Wood, 1L),
-                            200,
-                            8);
-                    GT_Values.RA.addCutterRecipe(
-                            new ItemStack(aStack.getItem(), 1, i),
-                            GT_Utility.copyAmount(
-                                    GT_Mod.gregtechproxy.mNerfedWoodPlank ? ((long) tStack.stackSize)
-                                            : ((long) (tStack.stackSize * 5 / 4)),
-                                    tStack),
-                            GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Wood, 2L),
-                            200,
-                            8);
-                    GT_ModHandler.addSawmillRecipe(
-                            new ItemStack(aStack.getItem(), 1, i),
-                            tPlanks,
-                            GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Wood, 1L));
+                    GT_Values.RA.stdBuilder().itemInputs(new ItemStack(aStack.getItem(), 1, i))
+                            .fluidInputs(Materials.Lubricant.getFluid(1L))
+                            .itemOutputs(
+                                    GT_Utility.copy(tPlanks),
+                                    GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Wood, 1L))
+                            .duration(10 * SECONDS).eut(8).addTo(sCutterRecipes);
+                    GT_Values.RA.stdBuilder().itemInputs(new ItemStack(aStack.getItem(), 1, i))
+                            .itemOutputs(
+                                    GT_Utility.copyAmount(
+                                            GT_Mod.gregtechproxy.mNerfedWoodPlank ? ((long) tStack.stackSize)
+                                                    : (((long) tStack.stackSize) * 5L / 4),
+                                            tStack),
+                                    GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Wood, 2L))
+                            .duration(10 * SECONDS).eut(8).addTo(sCutterRecipes);
```

Comment:
Should this have no fluid input? Wasn't this water? Because otherwise the machines could sometimes just do this recipes instead of the one with lubricant

Reply:
I just ra2ed that part :P
no fluid should generate the 3 water/distilled/lubricant recipes (its the rare case where RA2 is not explicit).
but that begs the question what the lubricant one is for. will remove that one